Packs/SplunkPy/Integrations/SplunkPy/README.md

```diff
@@ -524,14 +524,16 @@ Sends events to an HTTP event collector using the Splunk platform JSON event pro
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| event | The event payload key-value pair. An example string: "event": "Access log test message.". | Required |
+| event | The event payload key-value pair. An example string: "event": "Access log test message.". | Optional |
```

Comment:
A validation was added to make sure the argument will be required when not using the batch_events.